### PR TITLE
Add drag and drop install to hathor

### PR DESCRIPTION
**Pull request by wilsonge**
Pull Request for Issue #15026 .

### Summary of Changes
Fixes drag and drop install in Hathor

### Testing Instructions
Test drag and drop install via click upload a file and by actually dragging and dropping works

### Expected result
It installs

### Actual result
It does not install

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/templates/hathor/html/com_installer/install/default_form.php [hunk 1]
```diff
@@ -67,53 +68,94 @@
...
+            <input type="hidden" name="type" value="" />
+            <input type="hidden" name="installtype" value="upload" />
+            <input type="hidden" name="task" value="install.install" />
+            <?php echo JHtml::_('form.token'); ?>
+        </div>
+    </div>
+    </form>
 </div>
-</form>
+<div id="loading"></div>
```
Comment (C-Lodder) [inline administrator/templates/hathor/html/com_installer/install/default_form.php]: You deleted the closing `</form>` tag.

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 772c5c9d3b07044a842c6dd67be87092b6c03feb<br /><br />The component does successfully install when dragged and dropped but the Success message is not displayed - it is when you install using browse<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15116">issues.joomla.org/tracker/joomla-cms/15116</a>.</sub>

File: administrator/templates/hathor/html/com_installer/install/default_form.php (hunk 1 above)
Comment (wilsonge) [inline administrator/templates/hathor/html/com_installer/install/default_form.php]: No i didn't. It's two lines up :)

Comment (C-Lodder) [inline administrator/templates/hathor/html/com_installer/install/default_form.php]: oops

**Comment by wilsonge** [comment]
I don't see any difference to ISIS there?

**Comment by brianteeman** [comment]
Very true - didnt realise that - so thats a bug in both of them then - I will change my test here to a success and raise a seperate issue for that

**Comment by wilsonge** [comment]
Thanks :)

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 772c5c9d3b07044a842c6dd67be87092b6c03feb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15116">issues.joomla.org/tracker/joomla-cms/15116</a>.</sub>

**Comment by N6REJ** [comment]
Since hathor is specifically for folks with disabilities shouldn't we add those functions?
https://dev.opera.com/articles/accessible-drag-and-drop/

**Comment by brianteeman** [comment]
Which functions specifically! A lot of what that refers to is for users without fine motor control who cannot "drop" onto a small target which is not the case here

**Pull request merged**